web.config

```diff
@@ -10,36 +10,130 @@
+                    <add key="svg.br" value="image/svg+xml; charset=utf-8" />
+                    <add key="txt.br" value="text/plain; charset=utf-8" />
+                    <add key="xml.br" value="text/xml; charset=utf-8" />
+                    <add key="webmanifest.br" value="application/manifest+json; charset=utf-8" />
+                </rewriteMap>
+            </rewriteMaps>
             <outboundRules>
-                <rule name="X-UA-Compatible header">
-                    <match serverVariable="RESPONSE_X_UA_Compatible" pattern=".*" />
+                <!-- Restore the mime type for compressed assets. See below for more explanation -->
+                <rule name="RestoreMime" enabled="true">
+                    <match serverVariable="RESPONSE_Content_Type" pattern=".*" />
                     <conditions>
-                        <add input="{RESPONSE_CONTENT_TYPE}" pattern="^text/html" />
+                        <add input="{HTTP_URL}" pattern="\.((?:css|html|ico|js|map|svg|txt|xml|webmanifest)\.(gz|br))" />
+                        <add input="{CompressedExtensions:{C:1}}" pattern="(.+)" />
                     </conditions>
-                    <action type="Rewrite" value="ie=edge"/>
+                    <action type="Rewrite" value="{C:3}" />
+                </rule>
+                <!-- add vary header -->
+                <rule name="AddVaryContentEncoding" preCondition="PreCompressedBrotli" enabled="true">
+                    <match serverVariable="RESPONSE_Vary" pattern=".*" />
+                    <action type="Rewrite" value="Content-Encoding" />
+                </rule>
+                <!-- indicate response is encoded with brotli -->
+                <rule name="AddEncodingBrotli" preCondition="PreCompressedBrotli" enabled="true" stopProcessing="true">
+                    <match serverVariable="RESPONSE_Content_Encoding" pattern=".*" />
+                    <action type="Rewrite" value="br" />
                 </rule>
+                <!-- indicate response is encoded with gzip -->
+                <rule name="AddEncodingZopfli" preCondition="PreCompressedZopfli" enabled="true" stopProcessing="true">
+                    <match serverVariable="RESPONSE_Content_Encoding" pattern=".*" />
+                    <action type="Rewrite" value="gzip" />
+                </rule>
+                <preConditions>
+                    <preCondition name="PreCompressedFile">
+                        <add input="{HTTP_URL}" pattern="\.((?:css|html|ico|js|map|svg|txt|xml|webmanifest)\.(gz|br))" />
+                    </preCondition>
+                    <preCondition name="PreCompressedZopfli">
+                        <add input="{HTTP_URL}" pattern="\.((?:css|html|ico|js|map|svg|txt|xml|webmanifest)\.gz)" />
+                    </preCondition>
+                    <preCondition name="PreCompressedBrotli">
+                        <add input="{HTTP_URL}" pattern="\.((?:css|html|ico|js|map|svg|txt|xml|webmanifest)\.br)" />
```

Comment:
HTTP**S**_URL ?

Reply:
Nope, that's a predefined IIS server variable (all things between `{}` are)